src/core/lib/security/security_connector/local/local_security_connector.cc

```diff
@@ -178,23 +215,17 @@ grpc_security_status grpc_local_channel_security_connector_create(
         "Invalid arguments to grpc_local_channel_security_connector_create()");
     return GRPC_SECURITY_ERROR;
   }
-  // Check if local_connect_type is UDS. Only UDS is supported for now.
+  // Perform sanity check on UDS address.
```

Comment:
Please comment that TCP local connection check is during check_peer.

Reply:
Done.